server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -30,7 +30,7 @@ import net.javacrumbs.jsonunit.core.internal.Options
 import org.apache.http.HttpStatus.SC_OK
 import org.apache.james.GuiceJamesServer
 import org.apache.james.jmap.http.UserCredential
-import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ANDRE, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.jmap.rfc8621.contract.Fixture._
```

Comment:
Wildcard import.

Maybe it's worth a discussion, I think a wildcard import in a fixture object is legitimate...

Reply:
Would agree regarding a Fixture yes

Reply:
I just notice that the wildcard will be automatically added by IDE when using optimize import. Does this mean it is more optimized or bugged?